cabal.project

```diff
@@ -0,0 +1,18 @@
+repository cardano-haskell-packages
```

Comment:
This is just here to be future-proof in case we need sth from CHaP?

Reply:
Indeed.